### DISCUSSION TITLE
Please tell us which columns are included in the future test set

### DISCUSSION THREAD

**Discussion by RobinHan24** [PaddlePaddle/PaddleSpatial#52] [BaiduKDDCup2022] [answered]
Hi organizers, The following sentence is directly quoted from the report you provided - 'we still privately hold data of several months to evaluate the submitted models by participants. ' Does it mean that in the evaluate dataset, meteorological data will not be available or available? And how many data fileds in in the future test set? If there are 13 fields in the future test set which is the same as train dataset, so the value of meteorological data will be NAN? Please tell us which columns are included in the future test set? Thank you very much.

**Comment by zhoujingbo** [comment, marked as answer]
> Hi organizers, The following sentence is directly quoted from the report you provided - 'we still privately hold data of several months to evaluate the submitted models by participants. 
>' Does it mean that in the evaluate dataset, meteorological data will not be available or available?
 meteorological data will not be available

> And how many data fileds in in the future test set? If there are 13 fields in the future test set which is the same as train dataset, so the value of meteorological data will be NAN? Please tell us which columns are included in the future test set? Thank you very much.
An example about the test data can be found here: https://github.com/PaddlePaddle/PaddleSpatial/tree/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation
